internal/service/user.go

```diff
@@ -3,14 +3,15 @@ package service
 import (
 	"github.com/wizeline/CA-Microservices-Go/internal/entity"
 	"github.com/wizeline/CA-Microservices-Go/internal/logger"
+	"slices"
```

Comment:
Small fix. In Go is a good practice follow an importing order:  standard libraries, our libraries, third-party libraries